spec/models/dish_spec.rb

```diff
@@ -11,4 +43,11 @@
       it {should belong_to :chef}
   end
 
+  it "total_calories" do
+    expect(@dish4.ingredients).to eq([@ingredient4])
+    expect(@dish4.total_calories).to eq(@ingredient4.calories)
+    @dish4.ingredients << @ingredient8
+    expect(@dish4.ingredients).to eq([@ingredient4, @ingredient8])
+    expect(@dish4.total_calories).to eq(@ingredient4.calories + @ingredient8.calories)
+  end
 end
```

Comment:
Great `#total_calories` test.

I don't see any of the additional relationships that you set up in your models being tested here or in any other spec files, however.